package/Runtime/Scripts/Internal/NimbusIOSAdManager.cs

```diff
@@ -46,6 +51,8 @@ internal void OnAdRendered(string param)
 
         internal void OnError(string param)
         {
+            // TODO pass through error message like Android
+            // e.g 	var errMessage = adError.Call<string>("getMessage");
```

Comment:
@brugg Android exposes the error message that the Nimbus server sends back. Is that reflected in `param`? Is so great! If not, then onError should expose the Nimbus error message.